sdcm/cluster.py

```diff
@@ -630,6 +621,23 @@ def file_exists(self, file_path):
             self.log.error('Error checking if file %s exists: %s',
                            file_path, details)
 
+    @property
+    def is_enterprise(self):
+        if self._is_enterprise is None:
+            if self.is_rhel_like():
+                result = self.remoter.run("sudo yum search scylla-enterprise", ignore_status=True)
+                if 'One of the configured repositories failed (Extra Packages for Enterprise Linux 7 - x86_64)' in result.stdout:
```

Comment:
I would check exit status too

Reply:
`yum search scylla-enterprise` returns 0 if scylla-enterprise exists or not.

```
[amos@amos-centos7 ~]$ sudo yum search scylla-enterprise
Loaded plugins: auto-update-debuginfo, fastestmirror
Repository scylladb-scylla-3rdparty is listed more than once in the configuration
Loading mirror speeds from cached hostfile
 * base: mirrors.usc.edu
 * centos-sclo-rh: mirror.fileplanet.com
 * centos-sclo-sclo: repos.lax.quadranet.com
 * epel: d2lzkl7pfhq30w.cloudfront.net
 * epel-debuginfo: d2lzkl7pfhq30w.cloudfront.net
 * extras: mirror.fileplanet.com
 * updates: mirror.fileplanet.com
========================================================================================== N/S matched: scylla-enterprise ==========================================================================================
scylla-enterprise-debuginfo.x86_64 : Debug information for package scylla-enterprise
scylla-enterprise.x86_64 : Scylla is a highly scalable, eventually consistent, distributed, partitioned row DB.
scylla-enterprise-ami.noarch : Scylla AMI
scylla-enterprise-conf.x86_64 : Scylla configuration package
scylla-enterprise-jmx.noarch : Scylla JMX
scylla-enterprise-kernel-conf.x86_64 : Scylla configuration package for the Linux kernel
scylla-enterprise-server.x86_64 : The Scylla database server
scylla-enterprise-tools.noarch : Scylla Tools
scylla-enterprise-tools-core.noarch : Core files for Scylla tools

  Name and summary matches only, use "search all" for everything.
[amos@amos-centos7 ~]$ echo $?
0
[amos@amos-centos7 ~]$ sudo yum search scylla-enterprise-none
Loaded plugins: auto-update-debuginfo, fastestmirror
Repository scylladb-scylla-3rdparty is listed more than once in the configuration
Loading mirror speeds from cached hostfile
 * base: mirrors.usc.edu
 * centos-sclo-rh: mirror.fileplanet.com
 * centos-sclo-sclo: repos.lax.quadranet.com
 * epel: d2lzkl7pfhq30w.cloudfront.net
 * epel-debuginfo: d2lzkl7pfhq30w.cloudfront.net
 * extras: mirror.fileplanet.com
 * updates: mirror.fileplanet.com
Warning: No matches found for: scylla-enterprise-none
No matches found
[amos@amos-centos7 ~]$ echo $?
0
```

Reply:
ok